dpti/ti.py

```diff
@@ -919,6 +920,36 @@ def compute_task(job, inte_method, Eo, Eo_err, To, scheme="simpson"):
     return info
 
 
+def run_task(task_name, machine_file):
+    task_dir_list = glob.glob(os.path.join(task_name, "task.*"))
+    task_dir_list = sorted(task_dir_list)
+    work_base_dir = os.path.join(os.getcwd())
+    with open(machine_file) as f:
+        mdata = json.load(f)
+    machine = Machine.load_from_dict(mdata["machine"])
+    resources = Resources.load_from_dict(mdata["resources"])
+
+    submission = Submission(
+        work_base=work_base_dir,
+        resources=resources,
+        machine=machine,
+    )
+
+    task_list = [
+        Task(
+            command="ln -s ../../graph.pb graph.pb; lmp -in in.lammps",
```

Comment:
The command string in the `Task` object seems to be hardcoded. If possible, consider making this a parameter or a configurable option to increase flexibility. [medium]